apps/backoffice/package.json

```diff
@@ -15,6 +15,7 @@
     "coverage": "yarn coverage:b4f && yarn coverage:fe",
     "coverage:b4f": "vitest run --config ./vitest.config.node.ts --coverage",
     "coverage:fe": "vitest run --config ./vitest.config.react.ts --coverage",
+    "typecheck": "next build",
```

Comment:
In turboRepo config, `typecheck` step dependsOn `build` step, so i think with this config next app will be built two times....is there a way to add a typecheck step that "do nothing"?

Reply:
**question:** Have you tried with `"typecheck": "tsc --noEmit"` instead of `"typecheck": "next build"`?

Reply:
1) i don't know if you can set a step that does nothing (i will try and see) , but seeing the code-review log it does not seem like it gets done 2 times , plus the code-review does not call the build step , but maybe it's not connected ? , sorry i don't know a lot about turbo

2) i tried and it doesn't work as you can see in the "[test pr](https://github.com/pagopa/io-services-cms/pull/944)" , because of this inside a node-module(next/image-types)
`declare module '*.svg' {
  /**
   * Use `any` to avoid conflicts with
   * `@svgr/webpack` plugin or
   * `babel-plugin-inline-react-svg` plugin.
   */
  const content: any

  export default content
}`
that is needed for the .svg files

Reply:
i tried a "" for typecheck and i simply go back to the previous version and the as-is

Reply:
from what i see the build step is being done automatically only for whats inside `packages` and not whats inside `apps` even though the config should be the same , so i'm pretty sure the build is not being done twice but i don't know why this is happening , because in theory it should be as you say i think
<img width="347" alt="Screenshot 2024-07-12 alle 16 09 58" src="https://github.com/user-attachments/assets/f1fe448a-86f6-4668-8947-85c95f9c74f5">